### DISCUSSION TITLE
How to solve this Flet window size issue?

### DISCUSSION THREAD

**Discussion by TheKnight-Dev** [flet-dev/flet#2600] [Q&A]
### Question

Whenever I try to resize my flet window it just stays full screen!! Here is the code:
```
from flet import *

def main(page: Page):
page.title="Test"
page.window_height = 400
page.window_width = 400

if __name__ == "__main__":
app(main)
```
 It also gives me this error but the app runs ok except for the resizing issue:

`Gdk-CRITICAL **: 17:24:24.008: gdk_window_get_state: assertion 'GDK_IS_WINDOW (window)' failed`

Please Help and advise!! and Thanks in advance

OS: Ubuntu 23.10

### ------------------------------------------------------

- [X] I have searched for answers to my question both in the issues and in previous discussions.

**Comment by 09u2h4n** [comment]
I guess, the problem is gstreamer. You can reinstall it.

### Updating the system
`sudo apt update `

### Installing base gstreamer
```
sudo apt install libgstreamer1.0-dev libgstreamer-plugins-base1.0-dev
```
